account_statement_base_import/wizard/import_statement.py

```diff
@@ -23,98 +23,82 @@
 import os
 
 
-class CreditPartnerStatementImporter(orm.TransientModel):
+class CreditPartnerStatementImporter(models.TransientModel):
     _name = "credit.statement.import"
 
-    def default_get(self, cr, uid, fields, context=None):
-        if context is None:
-            context = {}
+    @api.model
+    def default_get(self, fields):
+        ctx = self._context
         res = {}
-        if (context.get('active_model', False) ==
-                'account.statement.profile' and
-                context.get('active_ids', False)):
-            ids = context['active_ids']
-            assert len(
-                ids) == 1, 'You cannot use this on more than one profile !'
-            res['profile_id'] = ids[0]
-            other_vals = self.onchange_profile_id(
-                cr, uid, [], res['profile_id'], context=context)
-            res.update(other_vals.get('value', {}))
+        if (ctx.get('active_model', False) == 'account.journal' and
+                ctx.get('active_ids', False)):
+            ids = ctx['active_ids']
+            assert len(ids) == 1, \
+                'You cannot use this on more than one journal !'
+            res['journal_id'] = ids[0]
+            values = self.onchange_journal_id(res['journal_id'])
+            res.update(values.get('value', {}))
         return res
 
-    _columns = {
-        'profile_id': fields.many2one('account.statement.profile',
-                                      'Import configuration parameter',
-                                      required=True),
-        'input_statement': fields.binary('Statement file', required=True),
-        'partner_id': fields.many2one('res.partner',
-                                      'Credit insitute partner'),
-        'journal_id': fields.many2one('account.journal',
-                                      'Financial journal to use transaction'),
-        'file_name': fields.char('File Name', size=128),
-        'receivable_account_id': fields.many2one(
-            'account.account', 'Force Receivable/Payable Account'),
-        'force_partner_on_bank': fields.boolean(
-            'Force partner on bank move',
-            help="Tic that box if you want to use the credit insitute partner "
-            "in the counterpart of the treasury/banking move."),
-        'balance_check': fields.boolean(
-            'Balance check',
-            help="Tic that box if you want OpenERP to control the "
-            "start/end balance before confirming a bank statement. "
-            "If don't ticked, no balance control will be done."),
-    }
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Import configuration parameter',
+        required=True)
+    input_statement = fields.Binary(
+        string='Statement file',
+        required=True)
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Credit institute partner')
+    file_name = fields.Char('File Name', size=128)
```

Comment:
String is optional as `file_name` field name will fill string value as `File Name`